R/create.R

```diff
@@ -114,6 +124,46 @@ create_project <- function(
   invisible(proj_get())
 }
 
+#' @rdname create_package
+#' @export
+create_quarto_project <- function(
+  path,
+  type = "default",
+  rstudio = rstudioapi::isAvailable(),
```

Comment:
`rstudio` does not seem used. Is this for future usage? 

`quarto::quarto_create_project()` won't create an RStudio project.

Reply:
Good point! I think I was thinking that we don't *need* to mark it as an RStudio Project since it will have `_quarto.yml`. But that's really getting ahead of myself and for now it makes more sense to handle more similar to what's done in `create_project()`